e2e/scripts/run-e2e.sh

```diff
@@ -82,4 +82,4 @@ test_dir="$(dirname $test_file)"
 
 # run the test file directly, this allows log output to be streamed directly in the terminal sessions
 # without needed to wait for the test to finish.
-go test -v "${test_dir}" --run ${ENTRY_POINT} -testify.m ^${TEST}$
+go test -v "${test_dir}" --run ${ENTRY_POINT} -testify.m ^${TEST}$ -timeout 20m
```

Comment:
believe this is still necessary? I think by default this is ten minutes?

Reply:
the new test can take a bit longer (~15m) this could maybe be updated in the future to be specific to a test?